json/src/lib.rs

```diff
@@ -835,7 +835,7 @@ impl<'a> serde::Serialize for ImportMultiRequestScriptPubkey<'a> {
                 #[derive(Serialize)]
                 struct Tmp<'a> {
                     pub address: &'a Address,
-                };
+                }
```

Comment:
This change is unrelated to this PR, perhaps it should be left out or done as a separate patch explaining that its just a clippy fix.